### DISCUSSION TITLE
Error: No module named 'webdriver_manager.utils'

### DISCUSSION THREAD

**Discussion by marques-raizen** [TobiasPankner/Teams-Auto-Joiner#185] [General]
I'm trying to run it on Mac Ventura 13.5.2 w/ Python 3 but i'm getting the following error:
`Traceback (most recent call last):
  File "/Users/leandro.marques/teams-Auto-Joiner/auto_joiner.py", line 15, in <module>
    from webdriver_manager.utils import ChromeType
ModuleNotFoundError: No module named 'webdriver_manager.utils'`

Any ideas what might be causing this?

**Comment by dixong** [comment]
The module reference needs to change to `from webdriver_manager.core.os_manager import ChromeType`

Once that's fixed there's a timeout data type error that's in Selenium 3, can be patched.

Worth noting that Selenium 3 is no longer being maintained and msedge is now avaliable in v4.

Although I could get Teams to load and login it looks like the format of the page has changed and no longer seems to work (very happy to be wrong)
